build_verify.sh

```diff
@@ -0,0 +1,77 @@
+#/usr/bin/env bash 
+
+
+fails=""
+
+# Invoke gofmt on the current directory.
+# -l flag causes it to only list files
+GOFMT_RESULTS=`gofmt -l .`
+if [ "$GOFMT_RESULTS" ]
+then
+	echo "FAIL: UNFORMATTED FILES:"
+	echo "GOFMT FINDS"
+	echo "$GOFMT_RESULTS"
+	fails="fails"
+fi
+
+
+# Find all files ending with .go that do not end with .peg.go (generated)
+GOLINT_FILES=`find . -not -name *.peg.go -name *.go`
+# Iterate over these (split by newline)
+
+# Check each of them with golint
```

Comment:
I don't think we should fail the build on `golint` failures, but we should run them.